README.md

```diff
@@ -48,6 +48,57 @@ required go version: 1.12
 go get github.com/jesseduffield/lazydocker
 ```
 
+### Docker
+
+1. <details><summary>Build it...</summary><p>
+
+    - If you have a x86_64 CPU architecture
+
+        ```sh
+        docker build -t lazydocker . https://github.com/jesseduffield/lazydocker.git
```

Comment:
Umm, `docker build` accepts only one argument: `PATH or URL or -`, so here maybe we should to split this command to two separate ones like:

`docker build -t lazydocker .`
> if building while being in source tree

` docker build -t lazydocker https://github.com/jesseduffield/lazydocker.git`
> if building without source tree

Reply:
Oh yes typo, my bad ! I guess it's easier for the user to build directly from the git repo, should we just keep it that way? The user probably knows how to git clone and then docker build the repo if he wants to take the long way I think.

Reply:
Well, personally I would keep two variants for user to choose. Looks like `docker build` is similar to `go install (locally cloned repo) | go get (pulling and installing)`.

Reply:
Yeah that's how it is now on my branch. However, with the Docker Hub build, it will pull the Docker image by itself when a `docker run` command is invoked, if the image does not exist so there is no need to build or even pull the image (except to update). I will leave the docker build instructions somewhere below the first few instructions, for developers/contributors because the average user should not need it 👍